### ISSUE TITLE
Queuries regarding the Azure Container connection string

### ISSUE THREAD

**Issue by amartyaroy1** [hyamchi/Blob-Triggered-Copy-Function-Using-Client-Libs#1]
Hi @hyamchi 
I was going through your code and would like to know the src_conn_string and destination_conn_string is it the connection string of a storage account only or its connection url for individual containers created within a storage account

**Comment by hyamchi** [comment]
Hi @amartyaroy1, those are the connection strings for the storage account only.

**Comment by amartyaroy1** [comment]
then y r u using 2 times connection string for the same storage account?

**Comment by amartyaroy1** [comment]
and how did you connect the function app to listen to the containers addition part?arent you using the function app url anywhere?

**Comment by hyamchi** [comment]
no, one is for source storage the other is for the destination,
for the 2nd question: it is using blob triggered function, the function is connected to the storage with the help of a trigger.  hope this help.

**Comment by amartyaroy1** [comment]
so basically you have 2 storage account created and in each storage account you have 1 container and you are trying to listen to the src storage account and based on the trigger you are copying the files from the src storage to the destination storage right?

Because your descrption suggested that you are using one storage account only not 2

**Comment by hyamchi** [comment]
you can review this for more info: 
https://medium.com/@yamchi/how-to-copy-a-blob-from-an-azure-storage-account-to-another-one-with-a-blob-triggered-function-efcae4b6edef

I also reference Microsoft links for more info in that article

**Comment by hyamchi** [comment]
yes your explanation is right
